src/XeroPHP/Remote/Response.php

```diff
@@ -213,14 +213,14 @@ public function findElementErrors($element, $element_index)
                 case 'ValidationErrors':
                     if (is_array($value)) {
                         foreach ($value as $error) {
-                            $this->element_errors[$element_index] = trim($error['Message'], '.');
+                            $this->element_errors[$element_index][] = trim($error['Message'], '.');
```

Comment:
I'm happy to merge the other functions, can you please clarify what's going on here?

Reply:
Sorry to answer now, this shouldn't have been in the same PR, somehow I commited to the same.
But to answer that: sometimes there's more than one error for the same $element_index, and I need to see them all, not only the last out of two (let's say) errors.

Reply:
Will this cause issues of that property is being read as a string?

Reply:
Even if this were to change, I think it would be best to consider this in a different PR. Keep history clean and PRs focused 👍